gtsam/discrete/DecisionTree.h

```diff
@@ -41,7 +41,6 @@ namespace gtsam {
    */
   template<typename L, typename Y>
   class DecisionTree {
-
```

Comment:
@varunagrawal this change is here only because your PR has not removed the superfluous decisiontree changes yet :-) Could you do that first before I (or you!) go over this PR?

Reply:
I definitely resolved those. :/